lib/jsonapi/utils/underscore.ex

```diff
@@ -0,0 +1,33 @@
+defmodule JSONAPI.Utils.Underscore do
+  @moduledoc """
+  Helpers for replacing underscores with dashes.
+  """
+
+  def underscore?, do: Application.get_env(:jsonapi, :underscore_to_dash, false)
+
+  def underscore(value) when is_atom(value) do
+    value
+    |> to_string
+    |> underscore
+  end
+
+  def underscore(value) when is_bitstring(value) do
```

Comment:
this should be `is_binary`

Reply:
Out of curiosity @jeregrine, why do you prefer `is_binary` over `is_bitstring`?  I've seen the reverse suggestions previously (on other projects).

Reply:
A bit string isn't QUITE a string or binary, as well as the docs use is_binary to select a binary string.

https://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html